src/widgets/refinement-list/__tests__/refinement-list-test.ts

```diff
@@ -88,6 +90,7 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/refinement-
         };
 
         renderWidget({ cssClasses });
+        // @ts-expect-error
```

Comment:
Why do we expect an error? Should we cast instead?

Reply:
yes, casting would be more correct, maybe with a todo comment? I mostly wanted quick wins in this PR. Can spend the extra minutes finding the best solution